File: retangulo/retangulo.java
import java.util.Scanner;

class Main {  
    public static void main(String args[]) {  
        Scanner input = new Scanner(System.in);
        System.out.println("Digite a altura: ");
        int altura = input.nextInt();
        System.out.println("Digite a largura: ");
        int largura = input.nextInt();

        System.out.printf("A área dsse retângulo é %d\n", altura * largura);

        input.close();
    }  
}

File: retangulo/retangulo.c
#include <stdio.h>

int main () {

    int altura, largura;

    printf("Digite a altura: ");
    scanf("%d", &altura);

    printf("Digite a largura: ");
    scanf("%d", &largura);

    printf("A área desse retãngulo é %d\n", altura * largura);

    return 0;
}